mod/primitives/pkg/encoding/hex/bytes.go

```diff
@@ -29,17 +29,23 @@ import (
 var ErrInvalidHexStringLength = errors.New("invalid hex string length")
 
 // EncodeBytes creates a hex string with 0x prefix.
-// Inverse operation is ToBytes or MustToBytes.
+// Inverse operation is ToBytes or ToBytesSafe.
 func EncodeBytes(b []byte) string {
 	hexStr := make([]byte, len(b)*2+prefixLen)
 	copy(hexStr, Prefix)
 	hex.Encode(hexStr[prefixLen:], b)
 	return string(hexStr)
 }
 
-// MustToBytes returns the bytes represented by the given hex string.
-// It panics if the input is not a valid hex string.
-func MustToBytes(input string) []byte {
+// ToBytesSafe returns the bytes represented by the given hex string.
+// If the input is not a valid hex string, it returns an empty byte slice.
+func ToBytesSafe(input string) []byte {
+	var bz []byte
+	defer func() {
+		if r := recover(); r != nil {
+			bz = []byte{}
+		}
+	}()
```

Comment:
I have renamed `MustToBytes` to `ToBytesSafe` to represent the behaviour accurately as it is no loner panic-ing on invalid hex string.